internal/dag/builder_test.go

```diff
@@ -188,7 +188,7 @@ func TestDAGInsertGatewayAPI(t *testing.T) {
 		Spec: gatewayapi_v1alpha1.GatewaySpec{
 			Listeners: []gatewayapi_v1alpha1.Listener{{
 				Port:     80,
-				Protocol: "HTTP",
+				Protocol: "ingress_http",
```

Comment:
Is this supposed to change? Seems like it should stay `HTTP`? Could we use a `const` from the gateway API package? https://github.com/kubernetes-sigs/gateway-api/blob/1caea70dce2571124a7a4887d32076ff8aa166e2/apis/v1alpha1/gateway_types.go#L255

Reply:
I can switch to those, I think this a find/replace error.